DAL/Migrations/20211201063709_RemakeToNullableTypeIntervID.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace DAL.Migrations
+{
+    public partial class RemakeToNullableTypeIntervID : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
+        }
```

Comment:
empty migration?

Reply:
Yes, because I thought that I change model in dal